src/index.html

```diff
@@ -1,13 +1,68 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="product-card">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin=""
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+
+  <body class="product-card__content">
+
+    <div class="card" data-qa="card">
+
+      <div class="card__photo">
+        <img
+          src="images/imac.jpeg"
+          height="134px"
+          width="160px"
+          alt="imac"
+        >
+      </div>
+
+      <div class="title">
+        <h2 class="title__info">
+            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </h2>
+        <div class="title__code">Код товара: 195434</div>
+      </div>
+
+      <div class="review">
+        <div class="review__stars">
+          <div class="review__item"></div>
+          <div class="review__item"></div>
+          <div class="review__item"></div>
+          <div class="review__item"></div>
+          <div class="review__item review__item--last_color"></div>
+        </div>
+        <div class="review__total">Отзывов: 5</div>
+      </div>
+
+      <h3 class="producte-price">
+        <div class="producte-price__left">Цена:</div>
+        <div class="producte-price__right">69 999 грн</div>
+      </h3>
```

Comment:
```suggestion
      <div class="producte-price">
        <h3 class="producte-price__left">
          Цена:
        </h3>
        <p class="producte-price__right">
          69 999 грн
        </p>
      </div>
```

h3 it's title which has text inside and div it's wrapper.